device-to-group-command.py

```diff
@@ -0,0 +1,390 @@
+'''
+esper app usage
+
+usage: esper_group_actions [-h]
+                            -c  {uninstall, whitelist, brightness, alarm_volume, ring_volume, music_volume, notification_volume, bluetooth, wifi, gps, ping, reboot}
+                            -g  GROUP_ID
+                            -v  VALUE
+
+example:
+
+./esper_group_actions -g 52ecfb3c-d1ad-4e66-8cf9-85daff8d7f3c -c ring_volume -v 50
+./esper_group_actions -g 52ecfb3c-d1ad-4e66-8cf9-85daff8d7f3c -c ping
+
+Runs on all active devices in a group. API key is required.
+'''
+
+import sys
+import requests
+import esperclient
+import time
+from esperclient.rest import ApiException
+
+import argparse
+active_device_list = []
+inactive_device_list = []
+
+#### Enterprise Configuration #######
+
+# Please fill the varibales under <>
+configuration = esperclient.Configuration()
+configuration.host = 'https://<endpoint-name>-api.shoonyacloud.com/api'
+configuration.api_key['Authorization'] = '<API-KEY>'
+configuration.api_key_prefix['Authorization'] = 'Bearer'
+enterprise_id = '<ENTERPRISE-ID>'
+configuration.group_per_page_limit = 5000  # int | Number of results to return per page. (optional) (default to 20)
+configuration.group_per_page_offset = 0    # int | The initial index from which to return the results. (optional) (default to 0)
+
+###############################
+
+def run_whitelist_command(device, package_name):
+    api_instance = esperclient.CommandsApi(esperclient.ApiClient(configuration))
+    command_args = esperclient.CommandArgs(package_name=package_name)
+    command = esperclient.CommandRequest(command='ADD_TO_WHITELIST', command_args=command_args)
+    try:
+        api_response = api_instance.run_command(enterprise_id, device.id, command)
+
+        if api_response.id is not None:
+            print('Whitelist command fired successfully on ' +  device.device_name + ' for ' + package_name)
```

Comment:
In Python3 there is a better way to format strings now - f-strings
```
print(f'Whitelist command fired successfully on {device.device_name} for {package_name}')
```

Reply:
done